

```
# This is formatted as code
$ 4 /3 * 5 ^ 3 x PI $
```



In [ ]:
PI = 3.14159

In [ ]:
radius_of_sphere = float(input("Enter the radius:"))

Enter the radius:5.6


In [ ]:
volume = 4 / 3 * PI * (radius_of_sphere **3)

In [ ]:
print(f'volume of sphere {volume:.2f}')

volume of sphere 735.62


$ 4/3\ x \  5.6 ^ 3\ x \ 3.14159 = 735.62 $

```
# This is formatted as code
```



$x_{1}$ + $2x_{2}$ - $x_{3}$ = -2

$2x_{1}$ + $ 7x_{2} $ - $ 8x_{3} $ = -16

$-2x_{2}$ + $2x_{3}$ = 2

Solutions

$x_{1}$ = -3

$x_{2}$ = 2

$x_{3}$ = 3

In [ ]:
# Calculating Compound interest
deposit = 100000;

In [ ]:
interest_rate = 12

In [ ]:
number_of_years = 5

In [ ]:
amount = deposit* (1 + interest_rate / 100)**number_of_years

In [ ]:
print(f'Amount:{amount:.2f}')

Amount:176234.17


# Calculating compound Interest



p = 100000

r = 12

t = 5

n = 1
amount = $ p (1 + \frac{r}{n} )^{nt} $

total amount = 176234.17